langchain/retrievers/weaviate_hybrid_search.py

```diff
@@ -57,19 +57,24 @@ def add_documents(self, docs: List[Document]) -> List[str]:
                 data_properties = {
                     self._text_key: doc.page_content,
                 }
+                if doc.metadata is not None:
+                    for key in doc.metadata.keys():
+                        data_properties[key] = doc.metadata[key]
+
                 _id = get_valid_uuid(uuid4())
                 batch.add_data_object(data_properties, self._index_name, _id)
                 ids.append(_id)
         return ids
 
-    def get_relevant_documents(self, query: str) -> List[Document]:
+    def get_relevant_documents(
+        self, query: str, where_filter: Optional[Dict[str, object]] = None
```

Comment:
we shouldn't change this method signature since it's defined on the parent class